hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoGetAccountInfoHandler.java

```diff
@@ -307,7 +307,8 @@ private StakingInfo getStakingInfo(
 
         final var stakedNode = account.stakedNodeId();
         final var stakedAccount = account.stakedAccountId();
-        if (stakedNode != null) {
+        if (stakedNode != null && stakedNode != -1) {
```

Comment:
nit : we can add a static final variable "SENTINEL_NODE_ID = -1" and use everywhere instead for clarity

Reply:
Defined both in StakeInfoHelper and used them in the code change.